data_preparation.py

```diff
@@ -136,7 +136,7 @@ def prepare_data_in_dir(input_dir, output_dir, rep_length, overlapping_reads, lo
                 merged_reads = extract_gz(merged_reads, output_dir=output_dir)
             split_fastq_file(fastq_file=merged_reads, output_dir=output_dir, cpu_count=cpu_count, max_memory=max_memory)
         else:
-            raise Exception(f"When using merge_opposing !")
+            raise Exception(f"There are not 2 files to merge - overlapping or partial overlapping reads.")
```

Comment:
i dont understand this message.. 
The user is confused and doesn't know what she did.. 
we must make it absolutely clear what should be done differently..

Reply:
sorry I feel this is still unclear..
how about:
"when parameter overlapping_reads is set to {overlapping_reads}, there should be 2 fastq files in input_dir but found only one! Consult docs for details."

Reply:
changed